web/src/components/troubleshoot/SupportBundleList.tsx

```diff
@@ -71,12 +64,22 @@ export const SupportBundleList = (props: Props) => {
   const history = useHistory();
   const {
     deleteBundleId,
-    setIsToastVisible,
     isToastVisible,
     toastMessage,
-    setIsCancelled,
+    toastType,
+    setIsToastVisible,
+    toastChild,
   } = useContext(ToastContext);
 
+  // rework this so full page refresh is not needed.
+  // const deleteBundleFromList = (deleteId: string) => {
```

Comment:
Page has to be refetched rather than filtered.
If we filter, it causes a test to fail when installing with CLI. I will work on this later on.